test/functional/answers_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AnswersControllerTest < ActionController::TestCase
```

Comment:
Are you not testing the create/destroy/update actions here because they're tested in unit tests? Do you think it wouldn't add enough to test the controller actions too? I think checking for the correct instance variables with:

```ruby
assert_not_nil assigns['answer'] # <= tests existence of @answer after action runs
```